WorkAndFun/src/main/resources/templates/announcement-form-update.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title th:text="#{announcement.title}"></title>
+    <link th:href="@{/css/form.css}" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+<nav>
+</nav>
+<h1 th:text="#{announcement.headline}"></h1>
+<div>
+    <form th:object="${announcement}" th:action="@{/edit/{id}(id=${announcement.id})}" method="post">
```

Comment:
```suggestion
    <form th:object="${announcement}" th:action="@{/edit/{id}(id=${announcement.id})}" th:method="put">
```
koniecznie przy użyciu wpisu thymeleaf'owego "th:", bo html'owy PUT nie istnieje